templates/projects.html

```diff
@@ -60,11 +61,30 @@
       margin: 0;
       font-weight: bold;
     }
+
+  #context-bar {
+    background-color: white !important;
```

Comment:
You're adding a context-bar and then having to override its style so it doesn't look like a context-bar. Better to not create a context-bar in the first place (see below).

Reply:
I think it is one half-dozen of the other. I was going for semantic consistency where the breadcrumb (and now action buttons) were always in the context bar and let the styling of the context bar change.

You're taking the path of displaying the same content (the breadcrumb) in the same location as the context bar bc it isn't styled the same. It's fine either way for the moment.